lib/internal/process/pre_execution.js

```diff
@@ -8,6 +8,14 @@ const {
   ObjectGetOwnPropertyDescriptor,
   SafeMap,
   StringPrototypeStartsWith,
+  Date,
+  DatePrototypeGetFullYear,
+  DatePrototypeGetMonth,
+  DatePrototypeGetDate,
+  DatePrototypeGetHours,
+  DatePrototypeGetMinutes,
+  DatePrototypeGetSeconds,
+  String,
```

Comment:
```suggestion
  Date,
  DatePrototypeGetFullYear,
  DatePrototypeGetMonth,
  DatePrototypeGetDate,
  DatePrototypeGetHours,
  DatePrototypeGetMinutes,
  DatePrototypeGetSeconds,
  ObjectGetOwnPropertyDescriptor,
  String,
  SafeMap,
  StringPrototypeStartsWith,
```

I think this will solve the conflict

Reply:
Thanks for the suggestion. I have resolved it manually.